crates/binstalk/src/fetchers/gh_crate_meta/hosting.rs

```diff
@@ -11,13 +11,17 @@ pub enum RepositoryHost {
     Unknown,
 }
 
+// Make sure to update NOVERSION_FILENAMES if you update
+// FULL_FILENAMES
 pub const FULL_FILENAMES: &[&str] = &[
     "{ name }-{ target }-v{ version }.{ archive-format }",
     "{ name }-{ target }-{ version }.{ archive-format }",
     "{ name }-{ version }-{ target }.{ archive-format }",
     "{ name }-v{ version }-{ target }.{ archive-format }",
 ];
 
+// Make sure to update FULL_FILENAMES if you update
+// NOVERSION_FILENAMES
```

Comment:
...no, I meant `FULL_FILENAMES` and the array in bins.rs: https://github.com/cargo-bins/cargo-binstall/pull/417/files#diff-0002e98001c66321c833796a2f4f9f44dbfcafcbc1941dd3fde081b171f6149bR57-R62

Reply:
Done